cinch/site.yml

```diff
@@ -29,6 +29,7 @@
   hosts: jenkins_master
   roles:
     - jenkins_master
+    - ntp
```

Comment:
I don't know if our goal is to keep slaves as tightly controlled as possible with minimal install other than Swarm or not. We previously have had ntp running on masters and not on slaves, but I put it in a separate role so that it can easily be added to slaves if we want. I could go either way on that.